### PR TITLE
v1.56.0 - Adding styles for the cuisineCollage.

### PR DESCRIPTION
**Pull request by SachaZvetelman**
This has been reviewed and approved on an internal PR already by @kevinrodrigues and @ashleynolan.

### CODE REVIEW

**Comment by sonnyprince** [comment]
@SachaZvetelman is this much different to `c-cuisinesWidget`?

**Comment by sonnyprince** [comment]
would also be good to have screenshots etc 😃

**Comment by SachaZvetelman** [comment]
@sonnyprince, I guess that's quite subjective 😄. Here's the screenshot for UK:

image

**Comment by dandel10n** [comment]
@SachaZvetelman you need to include the `_cuisines-collage.scss` file to fozzie/src/scss/_dependencies.scss

File: src/scss/components/optional/_cuisines-collage.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            width: 100%;
+            height: 330px;
+            display: flex;
+            flex-flow: row nowrap;
+            margin-bottom: spacing(x2);
+            justify-content: center;
+            align-items: flex-start;
+            align-content: normal;
+
+            button {
```
Comment (sonnyprince) [inline src/scss/components/optional/_cuisines-collage.scss]: could we target by class here, maybe `c-cuisinesCollage-button`?

File: src/scss/components/optional/_cuisines-collage.scss [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                width: 100%;
+                max-width: 461px;
+            }
+
+            button:first-child img {
+                margin-bottom: 5px;
+            }
+        }
+
+        @include media('<wide-huge') {
```
Comment (sonnyprince) [inline src/scss/components/optional/_cuisines-collage.scss]: Is it better to have this nested within line 65? Keeps an element and all responsive styles in one place.

File: src/scss/components/optional/_cuisines-collage.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            .c-cuisinesCollage-item:nth-child(1) {
+                display: none;
+            }
+        }
+
+        @include media('<mid-wide') {
```
Comment (sonnyprince) [inline src/scss/components/optional/_cuisines-collage.scss]: Same comment as above - better to keep styles together?

File: src/scss/components/optional/_cuisines-collage.scss [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        .c-cuisinesCollage-item:last-child {
+            margin-right: 0;
+
+            img {
+                height: 161px;
+                width: 100%;
+                max-width: 461px;
+            }
+
+            button:first-child img {
```
Comment (sonnyprince) [inline src/scss/components/optional/_cuisines-collage.scss]: Is there any other way to target this? quite specific

File: src/scss/components/optional/_cuisines-collage.scss (hunk 2 above)
Comment (sonnyprince) [reply src/scss/components/optional/_cuisines-collage.scss]: There's a few places throughout that may be better to re-shuffle and have the responsive styles live closer to the elements they are changing 🤔

**Pull request merged**